src/components/BorrowerProfile/SupporterDetails.vue

```diff
@@ -150,7 +150,6 @@ export default {
 			userCardStyleOptions: [
 				{ color: 'tw-text-action', bg: 'tw-bg-brand-50' },
 				{ color: 'tw-text-black', bg: 'tw-bg-brand-100' },
-				{ color: 'tw-text-action', bg: 'tw-bg-secondary' },
 				{ color: 'tw-text-white', bg: 'tw-bg-action' },
 				// { color: 'tw-text-800', bg: 'tw-bg-gray-50' },
```

Comment:
Looks like commented cases have more than 3 years ago, could we remove them as well?